tests/Backend/Workspaces/ParallelWorkspacesTestCase.php

```diff
@@ -44,12 +44,16 @@ protected function initTestWorkspace(
         array $options = [],
         bool $forceRecreate = false,
         bool $async = true,
+        Client $client = null
```

Comment:
tady sem dělal úpravy kvůli shared testům do kterých sem hrabal ale nedokončil, ale už sem to tu nechal bude sa to hodit.